benchmarks/cugraph/standalone/pylibcugraph_bench.py

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import time
+
+from cugraph_funcs import generate_edgelist
+
+# Call RMAT
+edgelist_df = generate_edgelist(scale=23, edgefactor=16)
+srcs = edgelist_df["src"]
+dsts = edgelist_df["dst"]
+weights = edgelist_df["weight"]
+weights = weights.astype("float32")
+
+print(f"num edges: {len(weights)}")
+print()
+
+########
+import cugraph
+
+st = time.time()
+G2 = cugraph.Graph(directed=True)
+print(f"cugraph Graph create time:      {time.time()-st}")
+G2.from_cudf_edgelist(edgelist_df, source="src", destination="dst",
+                      edge_attr="weight", renumber=True)
+st = time.time()
+result = cugraph.pagerank(G2, alpha=0.85, tol=1.0e-6, max_iter=500)
+print(f"cugraph time:      {time.time()-st}")
+
+########
+import pylibcugraph
+
+resource_handle = pylibcugraph.experimental.ResourceHandle()
+graph_props = pylibcugraph.experimental.GraphProperties(
+    is_symmetric=False, is_multigraph=False)
+st = time.time()
+G = pylibcugraph.experimental.SGGraph(
```

Comment:
```suggestion
G = pylibcugraph.experimental.MGGraph(
```

Reply:
I think this needs to stay `SGGraph` since we're not incorporating dask/MPI/etc. anywhere to distribute the data to separate `MGGraphs`.